test/keystore/registrationResponse.json

```diff
@@ -0,0 +1,11 @@
+{

```

Comment:
Rename this file to `registration_response.json` to align with the bitwarden key